File: payroll_app_backend/payroll_app_backend/Models/PaySlip.cs
namespace payroll_app_backend.Models
{
    using System;
    using System.Collections.Generic;
    using System.ComponentModel.DataAnnotations;
    using System.ComponentModel.DataAnnotations.Schema;
    using System.Data.Entity.Spatial;

    public partial class PaySlip
    {
        public int Id { get; set; }

        [Required]
        [StringLength(50)]
        public string FirstName { get; set; }

        [Required]
        [StringLength(50)]
        public string LastName { get; set; }

        [Required]
        [StringLength(50)]
        public string PayDate { get; set; }

        [Required]
        [StringLength(50)]
        public string PayFrequency { get; set; }

        public int AnnualIncome { get; set; }

        public int GrossIncome { get; set; }

        public int IncomeTax { get; set; }

        public int NetIncome { get; set; }

        public int SuperRate { get; set; }

        public int Pay { get; set; }
    }
}

File: src/app/pay-slip.ts
/*
    An Entity class that represents one Payslip
*/

export class PaySlip {
    constructor(
        public firstname: string,
        public lastname: string,
        public paydate: string,
        public payfrequency: string,
        public annualincome: string,
        public grossincome: string,
        public incometax: string,
        public netincome: string,
        public superrate: string,
        public pay: string
    ){}
}